### ISSUE TITLE
Database history incorrect

### ISSUE THREAD

**Issue by twixer84** [Tautulli/Tautulli#1167]
**Version:**
Version 1.4.25
**Branch:**
master
**Commit hash:**
cfd30996264b7e9fe4ef87f02d1cc52d1ae8bfca
**Operating system:**
Windows 7 Pro
**Python version:**
Python Version: 	2.7.14 (v2.7.14:84471935ed, Sep 16 2017, 20:19:30) [MSC v.1500 32 bit (Intel)]
**What you did?**
Histoy would not update. Did a Flush and after it logged history but now it shows an item was playing for 64 days and is messing up my stats
**What happened?**
Did a Flush and after it logged history but now it shows an item was playing for 64 days and is messing up my stats
**What you expected?**
It to fix my history. it must have made the DB a bit corrupt? 
**How can we reproduce your issue?**

You can if i give you my DB file
**What are your (relevant) settings?**
N/A
**Link to logs:**
https://imgur.com/sdzcgXB

**Comment by JonnyWong16** [comment]
You'll just have to delete that one entry.

**Comment by twixer84** [comment]
That was my plan but it does not show up. This all started after it stopped logging history. I ended up updating Plexpy and doing a Flush. After that I was able to log history but im guessing this was the one item that was stuck. I even have a DB editor for the Plexpy.db file and i was trying to look and see if there was a entry i manually could take out of the datebase but was not sure of the table it would be located in.
